config/api.yml

```diff
@@ -1129,6 +1129,11 @@
       :post:
       - :name: refresh_dialog_fields
         :identifier: svc_catalog_provision
+      - :name: delete
+        :identifier: dialog_delete
```

Comment:
just add this post action "delete" back to the collection and we should be good. Thanks